dashboard/test/controllers/registrations_controller_test.rb

```diff
@@ -189,6 +189,15 @@ class RegistrationsControllerTest < ActionController::TestCase
     assert_equal frozen_time + ' UTC', sign_in.sign_in_at.to_s
   end
 
+  test "update student without user param returns 400 BAD REQUEST" do
+    student = create :student
+    sign_in student
+    assert_does_not_create(User) do
```

Comment:
Copypasta, from the test case immediately below this one.  I mean, we definitely don't want this to create a user.  Should I remove this assertion from the test?